core/Plugin/Manager.php

```diff
@@ -1145,7 +1145,6 @@ private function reloadActivatedPlugin($pluginName, $pluginsToPostPendingEventsT
             && !Development::isEnabled()
             && $this->isPluginActivated('Marketplace')
             && $this->isPluginActivated($pluginName)) {
-
```

Comment:
Same indent problem here. Is some other rule going to fix this or are we going to have nested code same indent level as conditions above it?

Reply:
Yes. That's up next. `PSR12.ControlStructures.ControlStructureSpacing.CloseParenthesisLine` will move the `) {` to a new line.